common/script/index.coffee

```diff
@@ -755,7 +755,7 @@ api.wrap = (user, main=true) ->
         if user.balance < 1
           return cb? {code:401,message: i18n.t('notEnoughGems', req.language)}
         else
-          user.balance--
+          user.balance -= 1
```

Comment:
I knew you'd have a good reason, although I was not expecting that many. :) Your reasoning is very sound. No need to revert.